src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -22,37 +20,82 @@
     public OrderController() {
     }
 
+    private List<ProductDto> initializeSampleProducts() {
+        List<ProductDto> products = new ArrayList<>();
+        String[] names = {"Ubrania", "Kostmetyki", "Samochdy", "Elektronika"};
+        String descripiton = "Dummy item description.";
+        for (int id = 0; id < names.length; id++) {
+            products.add(new ProductDto(
+                    (long) (id + 1),
+                    names[id],
+                    descripiton,
+                    new BigDecimal(id*100),
+                    (long)id
+            ));
+        }
+        return products;
+    }
+
+    private List<OrderDto> generateOrders() {
+        List<OrderDto> orders = new ArrayList<>();
+        LocalDateTime purchaseDate = LocalDateTime.of(2019, Month.SEPTEMBER, 1, 10, 30, 0);
+        LocalDateTime deliveryDate = purchaseDate.plusDays(5);
+        List<ProductDto> products = initializeSampleProducts();
+        for (int i = 0; i < products.size(); i++) {
+            List<ProductDto> productsList = new ArrayList<>();
+            Long orderId = Long.valueOf(i + 1);
+            for (int j = 0; j <= i; j++) {
+                productsList.add(products.get(j));
+
+            }
+            orders.add(new OrderDto(orderId, 1L, 2L, purchaseDate, productsList, "DELIVERED", deliveryDate, orderValue(productsList)));
+        }
+        return orders;
+    }
+
+    private BigDecimal orderValue(List<ProductDto> products) {
+        BigDecimal output = new BigDecimal(0);
+
+        for (ProductDto product : products) {
+            output.add(product.getPrice());
+        }
+
+        return output;
+    }
+
+
+
     @RequestMapping(method = RequestMethod.GET)
     public List<OrderDto> getOrders(){
-        return OrderService.getOrders();
+        return generateOrders();
     }
 
     @RequestMapping(method = RequestMethod.GET, value = "/{orderId}")
     public OrderDto getOrderById(@PathVariable Long orderId) {
-        return OrderService.getOrders().stream()
+        return generateOrders().stream()
                 .filter(order -> order.getId().equals(orderId))
                 .findFirst()
                 .orElseThrow(() -> new NoSuchElementException("Order not found"));
     }
 
     @RequestMapping(method = RequestMethod.DELETE, value = "/{orderId}")
     public void deleteOrder(@PathVariable Long orderId){
-        OrderDto orderToDelete = OrderService.getOrders().stream()
+        OrderDto orderToDelete = generateOrders().stream()
                 .filter(product -> product.getId().equals(orderId))
                 .findFirst()
                 .orElseThrow(() -> new RuntimeException("Order non-existent"));
-        OrderService.getOrders().remove(orderToDelete);
+        generateOrders().remove(orderToDelete);
```

Comment:
metoda `generateOrders()` przy każdym wywołaniu buduje nową listę przykładowych zamówień, więc dodawanie do niej jakichś elementów lub ich usuwanie misja się z celem;
nie zmienia to jednak faktu, że metoda działa